phpBB/styles/prosilver/theme/forms.css

```diff
@@ -106,15 +94,21 @@ fieldset.fields1 .live-search div {
 	margin-bottom: 0;
 }
 
-/* Specific layout 2 */
-fieldset.fields2 dt {
+/* Specific layouts */
+fieldset.fields2 dt,
+fieldset.fields1 dt {
 	border-right-width: 0;
-	width: 15em;
+	width: 165px;
 }
 
-fieldset.fields2 dd {
+fieldset.fields2 dd,
+fieldset.fields1 dd {
 	border-left-width: 0;
-	margin-left: 16em;
+	margin-left: 176px;
+}
+
+fieldset.fields1 dd {
```

Comment:
Wouldn't it be cleaner to have the border-left-width definition in here instead of directly overwriting the margin-left for fieldset.fields1 dd from above it?

Reply:
Yeah saw that will fix it tonight